app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_of_command = list(command.split())
```

Comment:
would be better to use `command, copied_file, file_to_copy = command.split()` for split